pkg/operator/sync.go

```diff
@@ -244,6 +246,96 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 		return fmt.Errorf("refusing to read images.json version %q, operator version %q", imgs.ReleaseVersion, optrVersion)
 	}
 
+	// handle image registry certificates.
+	// parse these, add them to ctrlcfgspec and then handle these in the daemon write to disk function
+	cfg, err := optr.configClient.ConfigV1().Images().Get(context.TODO(), "cluster", metav1.GetOptions{})
```

Comment:
nit: generally speaking we try to use cached objects when possible via an informer, instead of directly doing `Get()`'s via API